packages/url_launcher/url_launcher/lib/src/types.dart

```diff
@@ -55,3 +55,15 @@ class WebViewConfiguration {
   /// Not all browsers support this, so it is not guaranteed to be honored.
   final Map<String, String> headers;
 }
+
+/// Additional configuration options for [LaunchMode.inAppBrowserView]
+@immutable
+class BrowserConfiguration {
+  /// Creates a new InAppBrowserConfiguration with given settings.
+  const BrowserConfiguration({this.showTitle = false});
+
+  /// Whether or not to show the webpage title when using Chrome Custom Tabs.
+  ///
+  /// Only works on Android.
```

Comment:
Same comments here as on the platform interface; platform details shouldn't be documented at this level.

Reply:
Made this comment platform-independent